editor/array_property_edit.cpp

```diff
@@ -49,11 +49,7 @@ Variant ArrayPropertyEdit::get_array() const {
 }
 
 void ArrayPropertyEdit::_notif_change() {
```

Comment:
It's not necessarily to do in this PR, but I guess we could remove `_notif_change` here and in `DictionaryPropertyEdit` and use `notify_property_list_changed` directly in the UndoRedo code which currently uses `_notif_change`.

Reply:
I am not sure if these classes are still in use, i have the feeling they are not. Probably in another PR they should be removed.

Reply:
Since now there are EditorProperty based editor for arrays and dictionaries.